junit5-extension/src/test/java/io/kroxylicious/testing/kafka/junit5ext/StaticFieldExtensionTest.java

```diff
@@ -18,15 +19,13 @@
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 
-import io.kroxylicious.testing.kafka.api.KafkaCluster;
-import io.kroxylicious.testing.kafka.common.BrokerCluster;
-import io.kroxylicious.testing.kafka.invm.InVMKafkaCluster;
+import java.util.concurrent.ExecutionException;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertInstanceOf;
 
 @ExtendWith(KafkaClusterExtension.class)
-public class StaticFieldExtensionTest extends AbstractExtensionTest {
+class StaticFieldExtensionTest extends AbstractExtensionTest {
 
     @Order(1)
```

Comment:
I'm not sure what the order annotations are actually trying to achieve.  My first thought was that it was a mechanism to ensure that the field where populated in a particular order, but the code ensures that that clusters are created before clients. 

As far as i can see, the @Order annotation  is actually have no effect.

Reply:
Maybe @tombentley knows the rationale for the `@Order` annotation in the test.

Reply:
I don't think its about controlling element creation/injection. Its controlling junit evaluation order, so I think its trying to say that if `1` fails there is no hope of anything else passing so evaluate `1` first to give a stable baseline. 

I don't know if it delivers anything in practice however (now we have a functional extension, it was probably quite helpful in bootstrapping everything).